tests/test_report.py

```diff
@@ -31,6 +31,42 @@ def test_report_html_defaults():
     os.remove(os.path.join(DEFAULT_REPORT_PATH, 'report.html'))
 
 
+def test_report_xml_defaults():
+    report = Report()
+    issue = Vulnerability(category='Test', issue_name='Test Issue', severity=Severity.VULNERABILITY, description='Test')
+    report.issues.add(issue)
+    report.generate_report_file(file_type='xml')
+    # We remove the issue we added to clean up after ourselves.
+    report.issues.remove(issue)
+    assert os.path.exists(os.path.join(DEFAULT_REPORT_PATH, 'report.xml'))
+    # We remove the report, to clean up after ourselves
+    os.remove(os.path.join(DEFAULT_REPORT_PATH, 'report.xml'))
+
+
+def test_report_csv_defaults():
+    report = Report()
+    issue = Vulnerability(category='Test', issue_name='Test Issue', severity=Severity.VULNERABILITY, description='Test')
+    report.issues.add(issue)
+    report.generate_report_file(file_type='csv')
+    # We remove the issue we added to clean up after ourselves.
+    report.issues.remove(issue)
+    assert os.path.exists(os.path.join(DEFAULT_REPORT_PATH, 'report.csv'))
+    # We remove the report, to clean up after ourselves
+    os.remove(os.path.join(DEFAULT_REPORT_PATH, 'report.csv'))
+
+
+def test_report_json_defaults():
+    report = Report()
+    issue = Vulnerability(category='Test', issue_name='Test Issue', severity=Severity.VULNERABILITY, description='Test')
+    report.issues.add(issue)
+    report.generate_report_file(file_type='json')
+    # We remove the issue we added to clean up after ourselves.
+    report.issues.remove(issue)
+    assert os.path.exists(os.path.join(DEFAULT_REPORT_PATH, 'report.json'))
```

Comment:
Should we see that the report outputs are correct?

Reply:
Yes, but I was going to wait until the reports were fully formed before doing so.

Reply:
I think we should have a TODO for this somewhere in the code so its documented, and we know (hopefully) when we are changing reports in the future.

Reply:
Agreed.

Reply:
Added a comment.